### DISCUSSION TITLE
DataChannel sample flutter app is not working

### DISCUSSION THREAD

**Discussion by zzaman88** [ant-media/Ant-Media-Server#5215] [General]
Hi dear,
I successfully built the Flutter sample app and tried to test the functionality but it's not working as expected.
For example, messages are not sent or received in apps .

I used the following data in apps:
Stream address: wss://test.antmedia.io:5443/WebRTCAppEE/websocket
Stream id: streamId_g3GLk9AVE

Used the following web app:
https://test.antmedia.io/WebRTCAppEE/datachannel.html?id=streamId_g3GLk9AVE

Could you please suggest what I am missing?

Thanks in advance!

**Comment by USAMAWIZARD** [comment]
can you please let me know which version of ant_media_flutter library are you using

**Comment by USAMAWIZARD** [comment]
am able to reproduce this issue, seems like there is a bug 
can you please create a new issue here https://github.com/ant-media/Ant-Media-Server/issues

**Reply by zzaman88** [reply]
I have created one:
https://github.com/ant-media/Ant-Media-Server/issues/5216

**Reply by USAMAWIZARD** [reply]
Thanks @zzaman88
